corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -475,6 +477,35 @@ Formplayer.ViewModels.EvaluateXPath = function() {
         };
         $.publish('formplayer.' + Formplayer.Const.EVALUATE_XPATH, [self.xpath(), callback]);
     };
+
+    self.matcher = function(flag, subtext) {
+        var match, regexp;
+        // Match text that starts with the flag and then looks like a path.
+        // CKEditor reserves the right to insert arbitrary zero-width spaces, so watch for those.
```

Comment:
I think part of this regex is useful to you. CKEditor is a rich text editor and it adds zero-width spaces for formatting sometimes. If you're using a text area you shouldn't need that.